### ISSUE TITLE
Butebo location view doesn’t seem to work

### ISSUE THREAD

**Issue by k8hughes** [devinit/data-hub-spotlights#142]
Butebo location view doesn’t seem to work

**Comment by k8hughes** [comment]
![Screenshot 2020-04-02 at 17 20 17](https://user-images.githubusercontent.com/36768381/78273095-440a4a00-7506-11ea-879e-89429cd981be.png)

**Comment by k8hughes** [comment]
@wakibi this is another one that could connect to a data issue, any chance you could check the data side and pop a note of this issue?

**Comment by edwinmp** [comment]
Can be fixed by #88

**Comment by k8hughes** [comment]
So I think we still have a problem here, and it might be a data problem and a UX problem.

Check this:  http://157.245.39.153/spotlight/spotlight-uganda?t=uganda_poverty_and_vulnerability&i=spotlight_on_uganda_2017.uganda_poverty_headcount&y=2014&lc=UG.d233&ln=butebo 

Any way you configuare the indictaors for this location you are going to get no data.  So what is the correct approach?  Should a location appear if we don't ever have data for it?
As a user I tried every drop down and got confused that nothing would work - maybe if we continue to let a user select a location where we have no data under any indicator we need to say there is no data?

@edwinmp  what do you think?

**Comment by edwinmp** [comment]
There is no data for Butebo, which is communicated by the map ... I think this the best & frankly only viable option we have here. Maybe to make it clearer, we can have the tooltip show by default, with 'No Data' (actually, this isn't a bad idea even for those with data, as long as we're viewing a single location)

I could create a CMS config that we use to excludes certain locations, but it can't account for clicking the excluded location on the map - someone may think it's a bug if we make them unclickable.

Bottom line, I'd say the locations should not be controlled by the data ... excluding it frankly seems a bigger bug than showing a user we have no data for it.

**Comment by benjamincoleman** [comment]
@edwinmp I like this idea: 

> Maybe to make it clearer, we can have the tooltip show by default, with 'No Data' (actually, this isn't a bad idea even for those with data, as long as we're viewing a single location)

As on touchscreens, there is no hover.

**Comment by k8hughes** [comment]
This looks good.
